docs/verify/simulate-with-qiskit-primitives.ipynb

```diff
@@ -0,0 +1,584 @@
+      "text/plain": [
+       "<Figure size 370.906x200.667 with 1 Axes>"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit import QuantumCircuit\n",
+    "from qiskit.circuit import Parameter\n",
+    "from qiskit.quantum_info import SparsePauliOp\n",
+    "\n",
+    "# circuit for which you want to obtain the expected value\n",
+    "qc = QuantumCircuit(2)\n",
+    "qc.ry(Parameter('theta'), 0)\n",
+    "qc.h(0)\n",
+    "qc.cx(0,1)\n",
+    "qc.draw(\"mpl\", style=\"iqp\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "e5beee91-315e-4f0e-b9dd-7a8faeb007a5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# observable(s) whose expected values you want to compute\n",
+    "observable = SparsePauliOp([\"II\", \"XX\", \"YY\", \"ZZ\"], coeffs=[1, 1, -1, 1])\n",
+    "\n",
+    "# value(s) for the circuit parameter(s)\n",
+    "import numpy as np\n",
+    "parameter_values = [[0], [np.pi/6], [np.pi/2]]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "97bf4aa1-189b-42e9-b77d-6d58f46386a5",
+   "metadata": {},
+   "source": [
+    "### Initialize an Estimator\n",
+    "\n",
+    "To use the Estimator V2 implementation, follow the instructions to instantiate a\n",
+    "[`qiskit.primitives.StatevectorEstimator`](../api/qiskit/qiskit.primitives.StatevectorEstimator). If you want to maintain\n",
+    "your pre-existing workflow using an Estimator V1 implementation, you can also use the\n",
+    "[`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) class.\n",
+    "\n",
+    "<Tabs>\n",
+    "  <TabItem value=\"EstimatorV2\" label=\"StatevectorEstimator (V2)\">\n",
+    "    Instantiate a [`qiskit.primitives.StatevectorEstimator`](../api/qiskit/qiskit.primitives.StatevectorEstimator).\n",
+    "\n",
+    "```python\n",
+    "from qiskit.primitives import StatevectorEstimator\n",
+    "estimator = StatevectorEstimator()\n",
+    "```\n",
+    "  </TabItem>\n",
+    "\n",
+    "  <TabItem value=\"EstimatorV1\" label=\"Estimator (V1)\">\n",
+    "    Instantiate a [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator).\n",
+    "\n",
+    "```python\n",
+    "from qiskit.primitives import Estimator\n",
+    "estimator = Estimator()\n",
+    "```\n",
+    "  </TabItem>\n",
+    "</Tabs>"
```

Comment:
Jupyter notebook doesn't accept "tabs" syntax, and these python cells would have to be converted to code cells, you can see that it's not rendered as intended:

![Screenshot 2024-04-08 at 10 21 01](https://github.com/Qiskit/documentation/assets/57907331/9d02a13b-c439-460f-97ac-fce468fa6e9c)

This means that we have no option but to show both V1 and V2 examples sequentially and add clear titles to indicate which one is V1 and which one is V2 (these were indicated through the tab). But if we want the examples to run (which I imagine is the motivation for the switch to notebooks), we'd also have to rename the variables because the V2 examples overwrite the variables from the V1 ones. This seems like a regression on the UX, to be honest, but it can be done, especially given that the notebook is quite short.

Reply:
Actually, it looks like, according to https://github.com/Qiskit/documentation/issues/1072, we should get rid of the V1 examples altogether. @kevinsung, have you been working on this independently or should this be done as part of this PR? It seems like an overkill to re-format the code examples if the V1 path is going to go away.